packages/debugger/lib/txlog/reducers.js

```diff
@@ -353,6 +369,7 @@ function currentNodePointer(state = "", action) {
     case actions.UNLOAD_TRANSACTION:
       return "";
     default:
+      //includes events
```

Comment:
what's the point of this comment? wouldn't one expect it to include events?

Reply:
I figured someone might think I forgot to include a case for events.  I'm making it explicit that they are intended to be part of the default case.

Reply:
ah, I see your reasoning!